code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -1078,8 +1080,8 @@
 		verbs -= /mob/living/silicon/robot/proc/choose_icon
 		return
 
-	if(icon_selection_tries == -1)
-		icon_selection_tries = module_sprites.len+1
+	if(icon_selection_tries == -1) // This is a bandaid solution to rig the 'borgs selection tries to 6
+		icon_selection_tries = 6 // If you know how to do it properly, please help.
```

Comment:
Could probably just do `icon_selection_tries = 2/3 * module_sprites.len` or similiar here.

Reply:
I had to try to find a way to round up since decimals/halves didn't work v. well with the icon select since it would only recognize the end of tries if you would reach -1, so I used round() to set it to round up, it's working a lot better now I think, thank you.